core/templates/dev/head/pages/collection_editor/editor_tab/CollectionLinearizerService.js

```diff
@@ -90,53 +54,26 @@ oppia.factory('CollectionLinearizerService', [
       return index;
     };
 
-    var getNodeLeftOf = function(linearNodeList, nodeIndex) {
-      return nodeIndex > 0 ? linearNodeList[nodeIndex - 1] : null;
+    var getNodeIndexLeftOf = function(linearNodeList, nodeIndex) {
+      return nodeIndex > 0 ? nodeIndex - 1 : null;
     };
 
-    var getNodeRightOf = function(linearNodeList, nodeIndex) {
+    var getNodeIndexRightOf = function(linearNodeList, nodeIndex) {
```

Comment:
Where is this used?

Reply:
Yeah, I had changed the function that used it. Removed it now.